tests/test_text_onnx_embeddings.py

```
@@ -308,3 +308,71 @@ def test_token_count(model_cache, model_name) -> None:
         doc_token_count = model.token_count(documents)
         assert first_doc_token_count + second_doc_token_count == doc_token_count
         assert doc_token_count == model.token_count(documents, batch_size=1)
+
+
+# ---------------------------------------------------------------------------
+# Regression tests for issue #688 – non-finite embedding guard
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `–` (EN DASH). Did you mean `-` (HYPHEN-MINUS)?

(RUF003)